packages/entity-detail/src/components/DetailForm/DetailForm.js

```diff
@@ -12,8 +12,10 @@ import readOnlyFormFieldMapping from '../../util/detailView/readOnlyFormFieldMap
 import StyledDetailForm from './StyledDetailForm'
 
 export class DetailForm extends React.Component {
-  componentWillReceiveProps(nextProps) {
-    this.props.fireTouched(nextProps.anyTouched)
+  componentDidUpdate(prevProps, prevState, snapshot) {
+    if (prevProps !== this.props) {
```

Comment:
what is the point of this `if`?

Reply:
standard componendDidUpdate check to control the calls, prevents infinite loops as well.